scripts/deploy/deployValidators.ts

```diff
@@ -10,7 +10,8 @@ async function main() {
   const chainId = hre.network.config.chainId;
 
   const stateContractAddress = getStateContractAddress();
-  const validators: ("mtpV2" | "sigV2" | "v3")[] = ["mtpV2", "sigV2", "v3"];
+  const validators: ("mtpV2" | "sigV2" | "v3" | "authV2")[] = [
+    "authV2"];
```

Comment:
Why only "authV2" validator now in the array?

Reply:
fixed